pkg/identity/base/issuer.go

```diff
@@ -20,9 +20,15 @@ import (
 	"regexp"
 	"strings"
 
+	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/sigstore/fulcio/pkg/identity"
 )
 
+var (
+	// For testing
+	CmpOptions = cmpopts.IgnoreUnexported(baseIssuer{})
```

Comment:
Just curious, what was the reason for this?

Reply:
when you use `cmp.Diff` you need to specify that unexported fields can be ignored when comparing two objects, otherwise the library panics. it's discussed a bit in the project [readme](https://github.com/google/go-cmp) but I'm using it for testing [here](https://github.com/sigstore/fulcio/pull/1008/files#diff-82a7a7c995f812a9240e539c523cf8f8c1b8484598447464beb9a0681e17a7ecR48)